File: Resueltos_por_competidor/Santiago_Ortiz/Football.java
import java.util.Scanner;

/*
 * Autor: Darkovski7   
 * Problema: Football
 * Juez Online: Codeforces
 * Veredicto: Accepted 
 * Url: https://codeforces.com/problemset/problem/96/A
 */

public class Football {

	static int cont = 1;

	public static void main(String[] args) {

		Scanner sc = new Scanner(System.in);

		String s = sc.next();

		for (int i = 0; i < (s.length() - 1); i++) {

			char actual = s.charAt(i);
			char siguiente = s.charAt(i + 1);

			if (cont == 7) {
				break;
			} else if (actual == siguiente) {
				cont++;
			} else {
				cont = 1;
			}
		}

		if (cont >= 7) {
			System.out.println("YES");
		} else if (cont < 7) {
			System.out.println("NO");
		}
	}
}

File: Resueltos_por_competidor/Carlos_Cobaleda/football.cpp
/*   
Autor: Carlos Eduardo Cobaleda Moreno
Problema: Football
Juez online: Codeforces
Veredicto: Accepted
URL: https://codeforces.com/contest/96/problem/A
*/

#include <bits/stdc++.h>
using namespace std;
 
int main() {
    string n; cin >> n;
    int count = 0;
    for (int i = 0; i < n.length(); i++) {
        if (n[i] == n[i-1]) {
            count++;
        } else {
            count = 1;
        }
 
        if (count >= 7) {
        break;
        }
    }
 
    if (count >= 7) {
    cout << "YES" << endl;
    } else {
        cout << "NO" << endl;
    }
return 0;
}
